packages/kit/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @sveltejs/kit
 
+## 2.5.8
+
+### Patch Changes
+
+- feat: handle `enctype` and `formenctype` properties for enhanced forms
```

Comment:
```suggestion
- fix: respect the form `enctype` and button `formenctype` attributes for enhanced forms
```

Reply:
Okay, although i am not sure how i should add this to changelog after i reverted the changes accordingly to [your comment](https://github.com/sveltejs/kit/pull/12198#discussion_r1596661619). Any recommendation?

Reply:
If you run `pnpm changeset` you should see a list of projects to select. Press `space` to select the `@sveltejs/kit` then `enter` to proceed to the next step. Next, you will be prompted if you would like to include it in a major bump, press `enter` to skip. Similarly for the minor bump prompt. Finally, you will reach a patch bump prompt. You can enter a summary of your change there "fix: ...". Completing this process creates a markdown file in the `.changeset` folder which will be included in the changelog for the next release.

Reply:
i've added a changelog entry using `pnpm changelog` according to your instructions ;)